lib/ui-switch/index.styl

```diff
@@ -68,5 +61,39 @@
       position absolute
       left 58%
       top 50%
-      transform translateX(-50%) translateY(-50%)
-      z-index 0
\ No newline at end of file
+      transform: translate(-50%, -50%);
+      z-index 0
+      white-space nowrap
+  
+  input[type="checkbox"]
+    &:checked + .status:after
+      content attr(data-label-true)
+      left 42%
+  
+  &.ui-switch-sm
+    width 90px
+    
+    input[type="checkbox"]
+      &:checked + .status:before
+        transform: translate(58px, -5%);
+      
```

Comment:
@Cherna Can we move these values to a variable? Meaning having a base size and modifying that value, just so its easier to change the size in the future.